storm-server/src/main/java/org/apache/storm/Testing.java

```diff
@@ -314,6 +308,21 @@ public static void withTrackedCluster(MkClusterParam param, TestJob code) {
         }
     }
 
+    /**
+     * In a tracked topology some metrics are tracked.  This provides a way to get those metrics.
+     * This is intended mostly for internal testing.
+     *
+     * @param id the id of the tracked cluster
+     * @param key the name of the metric to get.
+     * @return the metric
+     */
+    @SuppressWarnings("unchecked")
+    @Deprecated
+    public static int globalAmt(String id, String key) {
+        LOG.warn("Reading tracked metrics for ID {}", id);
+        return ((ConcurrentHashMap<String, AtomicInteger>) RegisteredGlobalState.getState(id)).get(key).get();
+    }
+
```

Comment:
Why was this moved?

Reply:
In order to keep the overloaded `withTrackedCluster` together